### DISCUSSION TITLE
How to update newly downloaded apps to the HK App.

### DISCUSSION THREAD

**Discussion by pmoney303** [maxileith/homebridge-appletv-enhanced#298] [Q&A] [answered]
I recently downloaded a new AppleTV app and it is not showing as one of the possible apps when creating automations with the Enhanced AppleTV.  How do I add, or for that matter remove, apps that are part of a TV already paired?

**Comment by maxileith** [comment, marked as answer]
Restart the plugin.

**Reply by maxileith** [reply]
@pmoney303 did that answer your question

**Reply by pmoney303** [reply]
I had restarted the plugin, it did not immediately resolve but did update several hours later.

**Reply by maxileith** [reply]
It refreshes immediately after the restart. It is technically impossible that it refreshes after running several hours. The problem is the Home app. You probably have not restarted the home app which is required to see the new apps. There is a lot of room for improvement on Apples side.
